polarify/main.py

```diff
@@ -182,9 +310,12 @@ def parse_body(full_body: list[ast.stmt], assignments: dict[str, ast.expr] | Non
         elif isinstance(stmt, ast.Return):
             if stmt.value is None:
                 raise ValueError("return needs a value")
-
             state.handle_return(stmt.value)
             break
+        elif isinstance(stmt, ast.Match):
+            if PY_39:
+                raise ValueError("match statements are only supported in Python 3.9+")
```

Comment:
```suggestion
            assert not PY_39
```

Reply:
Hello,

Thanks a lot for the suggestions!
I am currently in the middle of my finals, I will look over the comments after after Apr. 23rd.

Thanks,